libkbfs/keybase_daemon_rpc.go

```diff
@@ -305,3 +306,46 @@ func (k *KeybaseDaemonRPC) Shutdown() {
 		k.shutdownFn()
 	}
 }
+
+// GetTLFCryptKeys implements the TlfKeysInterface interface for
+// KeybaseDaemonRPC.
+func (k *KeybaseDaemonRPC) GetTLFCryptKeys(
+	ctx context.Context, tlfName string) (res keybase1.TLFCryptKeys, err error) {
+	var tlfHandle *TlfHandle
+
+getHandle:
+	for {
+		tlfHandle, err = ParseTlfHandle(ctx, k.config.KBPKI(), tlfName, false)
+		switch e := err.(type) {
+		case TlfNameNotCanonical:
+			tlfName = e.NameToTry
+		case nil:
+			break getHandle
+		default:
+			return res, err
+		}
+	}
+
+	res.CanonicalName = keybase1.CanonicalTlfName(tlfHandle.GetCanonicalName())
+
+	var rmd ImmutableRootMetadata
+	var id TlfID
+	_, rmd, id, err = k.config.KBFSOps().GetOrInitializeNewMDMaster(
+		ctx, k.config.MDOps(), tlfHandle)
+	if err != nil {
+		return res, err
+	}
+	res.TlfID = keybase1.TLFID(id.String())
```

Comment:
I meant specifically why is it being returned with this RPC call?  If you have some other use for it, adding a `GetTlfID` call makes sense. I don't care much one way or the other, I'm just wondering what it will be used for.